pkg/sql/codegen/tensorflow/template_train.go

```diff
@@ -39,11 +39,11 @@ from sqlflow_submitter.tensorflow.train import train
 from sqlflow_submitter.tensorflow.get_tf_version import tf_is_version2
 from tensorflow.estimator import DNNClassifier, DNNRegressor, LinearClassifier, LinearRegressor, BoostedTreesClassifier, BoostedTreesRegressor, DNNLinearCombinedClassifier, DNNLinearCombinedRegressor
 if tf_is_version2():
-    from tensorflow.keras.optimizers import *
-    from tensorflow.keras.losses import *
+    from tensorflow.keras.optimizers import Adadelta, Adagrad, Adam, Adamax, Ftrl, Nadam, RMSprop, SGD
```

Comment:
Use the same indention style as other `.py` files to improve readability?

Reply:
We are using 4 spaces indentation currently for all Python code and templates.